livereveal/main.js

```diff
@@ -106,8 +115,22 @@ function markupSlides(container) {
     
     // Put .end_space back at the end after all the rearrangement
     $('.end_space').appendTo('div#notebook-container');
+    return selected_cell_slide;
 }
 
+// Set the #slide-x-y part of the URL to control where the slideshow will start
+function setStartingSlide(selected) {
+    var start_slideshow = config.get_sync('start_slideshow_at');
+    if (start_slideshow === 'selected') {
+        // Start from the selected cell
+        window.location.hash = "/slide-"+selected[0]+"-"+selected[1];
```

Comment:
What do you think about using the Reveal API to do this?
https://github.com/hakimel/reveal.js#api, especially `Reveal.slide( indexh, indexv, indexf );` 
I think we should use this API as far as we can...